src/pages/auth/pages/kakao/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Wrapper } from './index.styles';
+
+export default function KakaoCallback() {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const queryParams = new URLSearchParams(window.location.search);
+    const accessToken = queryParams.get('access');
+    const refreshToken = queryParams.get('refresh');
+    const needOnboarding = queryParams.get('needOnboarding');
+
+    if (accessToken && refreshToken && needOnboarding) {
+      localStorage.setItem('access', accessToken);
+      localStorage.setItem('refresh', refreshToken);
+      localStorage.setItem('needOnboarding', needOnboarding);
+
+      if (needOnboarding === 'true') {
+        navigate('/onboarding');
+      } else {
+        navigate('/');
```

Comment:
그리고 로그인 완료 후 이전 로그인 화면으로 돌아갈 수 없게 replace를 사용해 주세요!
```suggestion
      if (needOnboarding === 'true') {
        navigate('/onboarding', ,  { replace: true });
      } else {
        navigate('/', ,  { replace: true });
```

Reply:
넵 수정했습니다~
감사합니다!